interfaces/builtin/system_packages_doc.go

```diff
@@ -40,6 +40,7 @@ const systemPackagesDocConnectedPlugAppArmor = `
 # Description: can access documentation of system packages.
 
 /usr/share/doc/{,**} r,
+/usr/local/share/doc/{,**} r,
```

Comment:
It would be slightly more efficient to change the rule on the line above to the following to combine these into a single rule:

```
/usr/{,local/}share/doc/{,**} r,
```